infra/modules/curi/mantarray_downloads/dns.tf

```diff
@@ -0,0 +1,71 @@
+resource "aws_route53_zone" "main" {
+  name = var.hosted_zone
+}
+
+resource "aws_route53_zone" "sub" {
+  name = "${var.subdomain}.${var.hosted_zone}"
+}
+
+
+resource "aws_route53_record" "ns_record" {
+  allow_overwrite = true
+  name            = var.hosted_zone
+  ttl             = 172800
+  type            = "NS"
+
+  zone_id = aws_route53_zone.main.zone_id
+  records = aws_route53_zone.main.name_servers
+}
+
+
+# resource "aws_route53_record" "sub_ns_record" {
+#   allow_overwrite = true
+#   name            = "${var.subdomain}.${var.hosted_zone}"
+#   ttl             = 172800
+#   type            = "NS"
+#   zone_id         = aws.route53_zone.sub.zone_id
+#   records         = aws_route53_zone.sub.name_servers
+# }
+
+
+resource "aws_route53_record" "a_record" {
+  zone_id = aws_route53_zone.main.zone_id
+  name    = var.hosted_zone
+  type    = "A"
+  ttl     = 3600
+
+  records = [
```

Comment:
make a comment that these are from Squarespace

Reply:
added